runtime/runtime/src/verifier.rs

```diff
@@ -403,7 +410,7 @@ pub fn validate_action(
         Action::Transfer(_) => Ok(()),
         #[cfg(feature = "protocol_feature_nonrefundable_transfer_nep491")]
         Action::NonrefundableStorageTransfer(_) => {
-            check_feature_enabled(ProtocolFeature::NonRefundableBalance, current_protocol_version)
+            check_feature_enabled(ProtocolFeature::PermanentStorageBytes, current_protocol_version)
```

Comment:
The `ProtocolFeature` was renamed but not the cargo feature. Doesn't matter much since the cargo feature will disappear eventually, but an inconsistency nevertheless.

Also I can't help but to notice that `NonrefundableStorage` in the action name sticks out like a sore thumb. How about `AllocatePermanentStorage`? It would also convey correctly that one can't "transfer" their own permanent storage to somewhere else...

Reply:
I would stick to the "transfer" name, as the action is essentially a transfer (just nonrefundable) and as a side effect it can create implicit account (as regular transfers can do). I can change the protocol feature name back to 'NonrefundableStorage` for consistency.